frontend/src/lib/components/SortableList/sortableList.tsx

```diff
@@ -0,0 +1,645 @@
+                        })
+                    ) {
+                        currentlyHoveredElement = null;
+                        setHoveredItemIdAndArea(null);
+                        return;
+                    }
+                    setHoveredItemIdAndArea({ id: hoveredElement.dataset.itemId ?? "", area });
+                    currentlyHoveredElement = {
+                        element: hoveredElement,
+                        id: hoveredElement.dataset.itemId ?? "",
+                        type: itemType,
+                        area,
+                        parentId: destinationId,
+                        parentType: destinationType,
+                    };
+                } else {
+                    currentlyHoveredElement = null;
+                    setHoveredItemIdAndArea(null);
+                }
+            }
+
+            function maybeCallItemMoveCallback() {
+                if (!onItemMoved) {
+                    return;
+                }
+
+                if (!draggedElement || !currentlyHoveredElement) {
+                    return;
+                }
+
+                if (isMoveAllowed !== undefined) {
+                    const parentElement = getItemParent(currentlyHoveredElement.element);
+                    const parentType = parentElement ? getItemType(parentElement) : null;
+                    if (
+                        !isMoveAllowed({
+                            movedItemId: draggedElement.id,
+                            movedItemType: draggedElement.type,
+                            originId: getItemParentGroupId(draggedElement.element),
+                            originType: getItemType(draggedElement.element),
+                            destinationId: getItemParentGroupId(currentlyHoveredElement.element),
+                            destinationType: parentType,
+                        })
+                    ) {
+                        return;
+                    }
+                }
+
+                if (
+                    currentlyHoveredElement.area === HoveredArea.HEADER ||
+                    currentlyHoveredElement.area === HoveredArea.CENTER
+                ) {
+                    const originId = getItemParentGroupId(draggedElement.element);
+                    const destinationId = currentlyHoveredElement.id;
+                    const position = 0;
+                    onItemMoved(draggedElement.id, originId, destinationId, position);
+                    return;
+                }
+
+                const originId = getItemParentGroupId(draggedElement.element);
+                const destinationId = getItemParentGroupId(currentlyHoveredElement.element);
+                const positionDelta = currentlyHoveredElement.area === HoveredArea.TOP ? 0 : 1;
+                const position = getItemPositionInGroup(currentlyHoveredElement.element) + positionDelta;
+
+                onItemMoved(draggedElement.id, originId, destinationId, position);
```

Comment:
If you move an item to the spots above or below itself, the item order wont actually change, but the `onItemMoved` event will still fire.

Wrote a simple check [for that in the welllog-viewer's code](https://github.com/equinor/webviz/blob/well-log-viewer/frontend/src/modules/WellLogViewer/settings/components/CurveTracks/TrackSettings.tsx#L122..L124), but that should prefferably be moved into this comp instead

```typescript
const currentPosition = props.plots.findIndex((p) => p.name === movedItemId);
if (currentPosition === newPosition || currentPosition + 1 === newPosition) return;
```